docs/changelog2.rst

```diff
@@ -0,0 +1,241 @@
+
+New features
+^^^^^^^^^^^^
+
+More informative repr with (truncated) WKT
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Support for fixed precision model for geometries and in overlay functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* `set_precision` to conform a geometry to a certain grid size, and this will then also be used by subsequent overlay methods
+* `grid_size` keyword in overlay methods (`intersection`, `union`, `difference`, etc) 
+
+
+* Addition of ``get_precision`` to get precision of a geometry and ``set_precision``
+  to set the precision of a geometry (may round and reduce coordinates) (#257)
+* Addition of ``grid_size`` parameter to specify fixed-precision grid for ``difference``,
+  ``intersection``, ``symmetric_difference``, ``union``, and ``union_all`` operations for
+  GEOS >= 3.9 (#276)
+
+Releasing the GIL
+~~~~~~~~~~~~~~~~~
+
+* Release the GIL to allow for multithreading in functions that do not
+  create geometries (#144) and in the STRtree ``query_bulk()`` method (#174)
+* Released the GIL in all geometry creation functions (#310, #326).
+
+STRtree improvements
+~~~~~~~~~~~~~~~~~~~~
+
+* Specifying a `predicate` in the STRtree queries 
+* Bulk queries
+
+
+* STRtree improvements for spatial indexing:
+  * Directly include predicate evaluation in ``STRtree.query()`` (#87)
+  * Query multiple input geometries (spatial join style) with ``STRtree.query_bulk`` (#108)
+* Fixed ``STRtree`` creation to allow querying the tree in a multi-threaded
+  context (#361).
+
+Bindings for new GEOS functionalities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Several (new) functions from GEOS are now exposed in Shapely:
+
+* ``haussdorff_distance`` and ``frechet_distance()``
+* ``contains_properly``
+* ``extract_unique_points``
+* ``reverse``
+* ``build_area()`` (GEOS >= 3.8)
+* ``minimum_bounding_circle`` and ``minimum_bounding_radius`` (GEOS >= 3.8)
+* ``coverage_union()`` and ``coverage_union_all()`` (GEOS >= 3.8)
+* ``segmentize`` (GEOS >= 3.10)
+* ``dwithin`` (GEOS >= 3.10)
+
+In addition some aliases for existing methods have been added to provide a
+method name consistent with GEOS or PostGIS:
+
+- ``line_interpolate_point`` (``interpolate``)
+- ``line_locate_point`` (``project``)
+- ``offset_curve`` (``parallel_offset``)
+- ``point_on_surface`` (``representative_point``)
+- ``oriented_envelope`` (``minimum_rotated_rectangle``)
+- ``delauney_triangles`` (``ops.triangulate``)
+- ``voronoi_polygons`` (``ops.voronoi_diagram``)
+- ``shortest_line`` (``ops.nearest_points``)
+- ``is_valid_reason`` (``validation.explain_validity``)
+
+
+Getting information / parts / coordinates from geometries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A set of GEOS getter functions are now also exposed to inspect geometries:
+`get_dimensions`, `get_coordinate_dimension`, `get_srid`, `get_num_points`,
+`get_num_interior_rings`, `get_num_geometries`, `get_num_coordinates`,
+`get_precision`.
```

Comment:
In theory this are also new functions exposing GEOS methods, like the section above. But it seemed worth to separate all those getters from actual spatial methods